app/javascript/app.spec.js

```diff
@@ -0,0 +1,14 @@
+import { shallowMount } from '@vue/test-utils';
+import App from 'app';
+
+describe('App', () => {
+  test('is a Vue instance', () => {
+    const wrapper = shallowMount(App);
+    expect(wrapper.isVueInstance()).toBeTruthy();
+  });
+
+  it('displays message on load', () => {
+    const wrapper = shallowMount(App);
+    expect(wrapper.find('p').text()).toEqual('Hello Vue!');
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'expect' is not defined    no-undef